Utilities/LongTerm/Check-Logs.ps1

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+param (
+    [string] $ConfigPath,
+    [string] $LogDestination,
+    [string] $ReportDestination
+)
+
+$SCRIPT_PATH = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$PARSER_PATH = Join-Path $SCRIPT_PATH "Get-Values.ps1"
+
+$SERVER_LOGS = @{"iostat" = "iostat.log"; "sar" = "sar_server.log";
+                 "mpstat" = "mpstat_server.log"; "sysbench" = "sysbench_run.log";
+                 "vmstat" = "vmstat_server.log"}
+$CLIENT_LOGS = @{"mpstat" = "mpstat_client.log"; "sar" = "sar_client.log";
+                 "vmstat" = "vmstat_client.log"}
+
+function Get-DataFromConfig {
+    param (
+        [string] $ConfigPath
+    )
+    # Expected config layout
+    # SERVER_IP={ip}
+    # SERVER_PORT={port}
+    # CLIENT_IP={ip}
+    # CLIENT_PORT={port}
+    # REMOTE_LOG_PATH={vmLogPath}
+    # VM_USERNAME={vmUser}
+    # VM_PASSWORD={vmPass}
+
+    $vmData = @{}
+    $content = Get-Content $ConfigPath
+
+    foreach ($line in $content) {
+        $key = $line.split("=")[0]
+        $value = $line.split("=")[1]
+
+        $vmData[$key] = $value
+    }
+
+    return $vmData
+}
+
+function Download-Log {
+    param (
+        [string] $vmIP,
+        [string] $vmPort,
+        [string] $vmUsername,
+        [string] $vmPassword,
+        [string] $remotePath,
+        [string] $destination
+    )
+
+    echo 'y' | & "C:\Program Files\PuTTY\pscp.exe" -P $vmPort -pw $vmPassword root@${vmIP}:${remotePath} $destination
```

Comment:
pscp directly, without full path.
or you cannot use the download function?

Reply:
sometimes pscp.exe cannot be found in path, so a full path is needed.
Will replace the path with $env:ProgramFiles so it's safer.
As for the download function, i don't consider it a good idea to import a module only for a function